config/locales/client.en.yml

```diff
@@ -4183,6 +4183,8 @@ en:
               wordpress: Necessary for the WordPress wp-discourse plugin to work.
             posts:
               edit: Edit any post or a specific one.
+            uploads:
+              create: Upload a new image.
```

Comment:
Change to "file" instead of "image" because discourse supports many different file types like .pdf or .mp4 and not just images.